packages/notifi-react-card/lib/utils/NotificationHistoryUtils.tsx

```diff
@@ -57,6 +62,32 @@ type SupportedEventDetailPropsMap = Map<
 
 const supportedEventDetails: SupportedEventDetailPropsMap = new Map();
 
+supportedEventDetails.set('DirectTenantMessageEventDetails', {
+  getViewProps: (notification: NotificationHistoryEntry) => {
+    const detail = notification.detail as DirectTenantMessageEventDetails;
+    const templateVariablesJson: Record<string, string> = JSON.parse(
+      detail.templateVariablesJson || '',
+    );
+    return {
+      notificationTitle: 'Announcement',
```

Comment:
is 'Announcement' right?

Do you have a screenshot

Reply:
@juni-notifi 
We do not have design for this. But I think it would be Announcement. Reasons below FYI:
- It is an announcement but only for particular wallets
- If we use directPush w/o specifying template , BE turns it to BroadcastMessageEventDetails .